### PR TITLE
459/feat/add slack util function for sending notifications

### PR DESCRIPTION
**Pull request by kodtld**

### CODE REVIEW

File: src/components/ConfirmCard/index.tsx [hunk 1]
```diff
@@ -75,17 +93,49 @@ export function ConfirmCard({
...
+                    checked={isNotifyChecked}
+                    onChange={(event) => {
+                      setIsNotifyChecked(event.target.checked);
+                      onNotifyChange && onNotifyChange(event.target.checked);
+                    }}
+                  />
+                }
+              />
+            </FormGroup>
+          )}
```
Comment (erjavaskivuori) [inline src/components/ConfirmCard/index.tsx:130]: On smaller screens this checkbox and its label move on the side of cancel and confirm buttons and is therefore partially invisible.

**Review by erjavaskivuori — CHANGES_REQUESTED** [review]
Commented about an issue with responsiveness and there is some merge conflict with staging but otherwise everything is looking good to me.

File: src/components/ConfirmCard/index.tsx (hunk 1 above)
Comment (kodtld) [reply src/components/ConfirmCard/index.tsx:133]: Done :+1:

**Review by erjavaskivuori — APPROVED** [review]
Looks good to me! :shipit:

**Pull request merged**